types/chromecast-caf-receiver/cast.framework.messages.d.ts

```diff
@@ -1100,6 +1128,11 @@ export class QueueItem {
      */
     playbackDuration?: number | undefined;
 
+    /**
+     * Used to track original order of an item in the queue to undo shuffle.
+     */
+    orderId?: number | undefined;
```

Comment:
where does this one come from?:
https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.media.QueueItem?hl=en

Reply:
https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.QueueItem?hl=en#orderId
?

Reply:
The first link you sent is related to a different package, `chromecast-caf-sender` . The correct reference is the second link, which contains the attribute.